src/Command/ExplainAdminCommand.php

```diff
@@ -34,14 +32,11 @@ final class ExplainAdminCommand extends Command
     private $pool;
 
     /**
-     * @var MetadataFactoryInterface
+     * @internal This class should only be used through the console
```

Comment:
should this annotation be on the constructor?

Reply:
Imho only the constructor should be internal. Adding this to the whole would say we can change the behavior of the command